src/rqt_image_view/image_view.cpp

```diff
@@ -75,9 +75,15 @@ void ImageView::initPlugin(qt_gui_cpp::PluginContext& context)
 
   connect(ui_.dynamic_range_check_box, SIGNAL(toggled(bool)), this, SLOT(onDynamicRange(bool)));
 
-  ui_.save_as_image_push_button->setIcon(QIcon::fromTheme("image-x-generic"));
+  std::string filepath = ros::package::getPath("rqt_image_view")+"/icons/disk_icon.png";
```

Comment:
Please use an icon provided by the theme instead of embedding your own: e.g. `document-save-as` (see https://github.com/ros-visualization/rqt_dep/blob/013475c07c066c34dcfc4e32d3c9ee29a15968ba/src/rqt_dep/ros_pack_graph.py#L171).